tests/tools/test_genpdf_export.py

```diff
@@ -52,3 +52,9 @@ def test_dump_blocks(tmp_path):
                     continue
                 np.testing.assert_allclose(v, blocks2[0][k])
             _pdf = lhapdf.mkPDF("new_pdf", 0)
+        for x in new_blocks[0]["xgrid"][1:-1]:
+            for q2 in new_blocks[0]["Q2grid"]:
+                data_from_block = new_blocks[0]["data"][new_blocks[0]["Q2grid"].index(q2)+len(new_blocks[0]["Q2grid"])*list(new_blocks[0]["xgrid"]).index(x)][6]
+                data_from_pdf = _pdf.xfxQ2(21,x,q2)
+                np.testing.assert_allclose(data_from_block,data_from_pdf, rtol = 1e-10)
```

Comment:
it seems you're not running pre-commit ... please install and run

Reply:
Eeehm maybe you mislabelled https://github.com/N3PDF/banana/pull/23/commits/47aa931d93905f48a57853eabe56a87e3163ffbc ? since pre-commit (i.e. specific black) didn't run still ... However, you introduced a bunch of `.DS_STORE` which, I believe is a Mac thing and not needed - please remove again (and add to  `.gitignore` if necessary)

Reply:
Ok now it should be fine :)